rdf_skos.services.yml

```diff
@@ -15,3 +15,6 @@ services:
     arguments: ['@entity_type.manager', '@rdf_skos.sparql.graph_handler']
     tags:
       - { name: event_subscriber }
+  rdf_skos.op_skos_setup:
```

Comment:
The service name should be updated, maybe `rdf_skos.skos_graph_configurator`?

Reply:
we agreed on rdf_skos.graph_configurator